pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -254,6 +262,17 @@ type SupportedRuntime struct {
 	Spec ServingRuntimeSpec
 }
 
+// WorkerSpec is the schema for multi-node/multi-GPU feature
+// If WorkerSpec is specified, it is assumed that multi-node, multi-gpu will be used.
+// For now, it is specifically for vllm using ray
+type WorkerSpec struct {
+	ServingRuntimePodSpec `json:",inline"`
```

Comment:
`containers` are actually mandatory field, but I think for most of the cases KServe is going to set this field from serving runtime

Reply:
Yes right but it is still possible to be set by isvc as well.